src/node/extension/extension.ts

```diff
@@ -388,15 +386,31 @@ function startSession(config: any): StartSessionResult {
 		}
 	}
 
+	let determineDebugTypeP: Promise<string|null>;
+	if (config.request === 'attach' && typeof config.processId === 'string' && config.processId.trim() === '${command:PickProcess}') {
+		determineDebugTypeP = pickProcessAndDetermineDebugType(config).then(debugType => {
+			if (debugType) {
+				config.processId = undefined;
+				config.port = debugType === 'node2' ? INSPECTOR_PORT_DEFAULT : LEGACY_PORT_DEFAULT;
+			}
+
+			return debugType;
```

Comment:
When running two node 8 processes, sending SIGUSR1 to the second results in the (expected) message: "Starting inspector on 127.0.0.1:9229 failed: address already in use". As a consequence "lsof/netstat" cannot determine the port and the "debugType" above will be null. Currently there is no error message shown in VS Code (and the node 8 message is probably not always visible). So we should consider to improve the error reporting in the "startSession" command.